gatt_blacklist.txt

```diff
@@ -23,6 +23,11 @@
 # Block access to standardized unique identifiers, for privacy reasons.
 00002a25-0000-1000-8000-00805f9b34fb
 
+# Random UUID for testing a characteristic excluding reads.
+# Should be removed from this blacklist if a characteristic excluding reads
+# is added.
+bad1c9a2-9a5b-4015-8b60-1579bbbf2135 exclude-reads
```

Comment:
I was thinking we'd add a temporary exclude-reads UUID to our hard-coded implementation to help test it, not that we'd add it to the public blacklist. We won't need these for testing purposes anyway once we have tests for updating the blacklist, so it seems like an implementation detail rather than something that needs to be public.